nucliadb_models/requirements.txt

```diff
@@ -1,2 +1,4 @@
 pydantic==1.9.0
-protobuf
\ No newline at end of file
+protobuf
+nucliadb-ingest>=1.0.0
+nucliadb-protos>=0.0.29
```

Comment:
It feels wrong that nucliadb_models is depending on nucliadb_ingest and nucliadb_protos.

Reply:
ah I see, it's because of the serialization logic

Reply:
This should remove the models -> ingest dependency:

https://github.com/nuclia/nucliadb/pull/291